data/state.js

```diff
@@ -67,9 +67,14 @@ export default function reducer (state = defaultState, action) {
 					else return Object.assign({}, state, { isGift: action.isGift })
 
 		case actions.EMAIL_ADDRESS_CHANGE:
-					return Object.assign({}, state, {
-						emailAddresses: state.emailAddresses.map((x, i) => i === action.index ? action.value : x)
-					})
+					const maxItems = state.isGift ? state.credit : 10;
+					const changed = action.value.split(',')
```

Comment:
maybe add a comment here about users comma-separating addresses, just so it's clear why this code exists